tests/test_smoke.py

```diff
@@ -3374,11 +3374,15 @@ def test_gcp_disk_tier():
         name_on_cloud = common_utils.make_cluster_name_on_cloud(
             name, sky.GCP.max_cluster_name_length())
         region = 'us-west2'
+        instance_type_option = ''
+        if disk_tier == resources_utils.DiskTier.BEST:
+            # Ultra disk tier requires a specific instance type.
```

Comment:
Please confirm:
```suggestion
            # Ultra disk tier requires n2 instance types to have more than 64 CPUs.
```

Reply:
Also, could we also test that with `BEST` and the instance type not supporting it, i.e. we are using the correct `high` tier for those instance types?

Reply:
Done! And the smoke test passed as well.